From a8e6251c05f1e956bdd24339b40d42255d73600b Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 18 Dec 2019 14:29:23 +0200
Subject: [PATCH] Revert "[SW-0] gaudi: pcie: skip preboot_init_pcie() at mmNIC2_HBM_PLL_BASE WA"

Since we remove the PLL W/A at all (no more old SoCs - only MF), we no
longer need to take special care about the W/A and PCIe PLL init.
Remove the code.

This reverts commit 40e395e0991d2a77c7e33399f5de6f4ef9327dea.

Change-Id: I180f9cebc9d57fd0054f4022e2a92ed09acb1057
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/common.c b/gaudi/common.c
index 2d5174d..29dc7aa 100644
--- a/gaudi/common.c
+++ b/gaudi/common.c
@@ -113,10 +113,6 @@
 	if (reset_cause == LKD_HARD_RESET_MAGIC)
 		return;
 
-	/* Skip if this is part of mmNIC2_HBM_PLL_BASE workaround */
-	if (regmap_readl(mmPSOC_GLOBAL_CONF_COLD_RST_FLOPS_2) == 1)
-		return;
-
 	preboot_init_pcie();
 }
 
